specifyweb/frontend/js_src/lib/components/Header/NotificationRenderers.tsx

```diff
@@ -164,22 +163,22 @@ export const notificationRenderers: IR<
   'record-merge-succeeded'(notification) {
     const id = parseInt(notification.payload.new_record_id);
     const tableName = notification.payload.table;
-    const model = getModel(tableName);
-    const resource = React.useMemo(
-      () =>
-        typeof model === 'object' ? new model.Resource({ id }) : undefined,
-      [model, id]
-    );
+    const newRecordId = parseInt(notification.payload.new_record_id);
```

Comment:
could use use `Number.parseInt` in these two places here instead of `parseInt`? thanks

eslint: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-number-properties.md